package.json

```diff
@@ -11,6 +11,7 @@
     "@testing-library/jest-dom": "^5.11.8",
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.6.0",
+    "nivo": "^0.31.0",
```

Comment:
Is there a reason for installing `nivo`? `@nivo` is already installed, and `nivo` is the (deprecated?) NPM package for installing Nivo.